com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/EditorDefaultSettings.cs

```diff
@@ -11,47 +12,30 @@ static class EditorDefaultSettings
         /// <returns>The default VolumeProfile if an HDRenderPipelineAsset is the base SRP asset, null otherwise.</returns>
         internal static VolumeProfile GetOrAssignDefaultVolumeProfile()
         {
-            if (!(GraphicsSettings.renderPipelineAsset is HDRenderPipelineAsset hdrpAsset))
-                return null;
-
-            return GetOrAssignDefaultVolumeProfile(hdrpAsset);
+            return HDDefaultSettings.instance.GetOrCreateDefaultVolumeProfile();
```

Comment:
so i was checking the file, and it is not needed anymore actually. but i am afraid to remove it and break people's project because they could rely on it, what do you think?

Reply:
after chatting with Alex, we decided to render these Obsolete just in case and we ll remove later